react-app/src/components/ResultSidebar.js

```diff
@@ -28,20 +27,14 @@ function ResultSidebar(props) {
       if (log.level === 'warn') {
         warnings += 1;
       }
-      if (log.level === 'debug') {
-        debugs += 1;
-      }
       total += 1;
     });
 
     return (
       <Accordion.Body>
-        <p key={0}>Errors: {errors}</p>
-        <p key={1}>Warnings: {warnings}</p>
-        <p key={2}>Debug Messages: {debugs}</p>
-        <p className="emphasized-list-text" key={3}>
-          Total Messages: {total}
-        </p>
+        <p className="emphasized-list-text">Total Messages: {total}</p>
```

Comment:
Just writing down what we talked about in the working session, but I think we can remove the total count and just keep the total error and warning messages.